### PR TITLE
[ENH]: Consistent `DataGrabber` naming in `testing`

### PR DESCRIPTION
**Pull request by synchon**
* [x] description of feature/fix
* [x] tests added/passed
* [x] add an entry for the latest changes

This PR renames the datagrabbers to `*DataGrabber` in `junifer.testing` to be consistent with the generally available ones in `junifer.datagrabber`.

### CODE REVIEW

File: junifer/testing/datagrabbers.py [hunk 1]
```diff
@@ -14,8 +14,12 @@
 from ..datagrabber.base import BaseDataGrabber
 
 
-class OasisVBMTestingDatagrabber(BaseDataGrabber):
-    """Data Grabber for Oasis VBM testing data."""
+class OasisVBMTestingDataGrabber(BaseDataGrabber):
+    """Data Grabber for Oasis VBM testing data.
```
Comment (LeSasse) [inline junifer/testing/datagrabbers.py:18]: Should there be a whitespace here in the docstring i.e. `Data Grabber` or `DataGrabber`?

File: junifer/testing/tests/test_oasisvmbtesting_datagrabber.py [hunk 2]
```diff
@@ -3,10 +3,10 @@
 # Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
 # License: AGPL
 
-from junifer.testing.datagrabbers import OasisVBMTestingDatagrabber
+from junifer.testing.datagrabbers import OasisVBMTestingDataGrabber
 
 
-def test_OasisVBMTestingDatagrabber() -> None:
+def test_OasisVBMTestingDataGrabber() -> None:
     """Test Oasis VBM Testing datagrabber."""
```
Comment (LeSasse) [inline junifer/testing/tests/test_oasisvmbtesting_datagrabber.py:10]: Should it be `DataGrabber` in the docstring here?

File: junifer/testing/tests/test_spmauditory_datagrabber.py [hunk 3]
```diff
@@ -3,10 +3,10 @@
 # Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
 # License: AGPL
 
-from junifer.testing.datagrabbers import SPMAuditoryTestingDatagrabber
+from junifer.testing.datagrabbers import SPMAuditoryTestingDataGrabber
 
 
-def test_SPMAuditoryTestingDatagrabber() -> None:
+def test_SPMAuditoryTestingDataGrabber() -> None:
     """Test SPM Auditory datagrabber."""
```
Comment (LeSasse) [inline junifer/testing/tests/test_spmauditory_datagrabber.py:10]: `datagrabber` -> `DataGrabber`?

**Review by LeSasse — CHANGES_REQUESTED** [review]
I think there also some changes that could be applied in the tests of the datagrabber module. For example, under `junifer/datagrabber/aomic/tests` in `test_id1000.py` there are some functions like `test_id1000_datagrabber`. Should they be renamed as well? I am not sure since in that instance I guess they don't refer to a class name explicitly, but might be worth reviewing that sub-package. (example here https://github.com/juaml/junifer/blob/main/junifer/datagrabber/aomic/tests/test_id1000.py#L13)

File: junifer/testing/datagrabbers.py (hunk 1 above)
Comment (synchon) [reply junifer/testing/datagrabbers.py:18]: For consistency in code, maybe it's better to remove the whitespace.

File: junifer/testing/tests/test_oasisvmbtesting_datagrabber.py (hunk 2 above)
Comment (synchon) [reply junifer/testing/tests/test_oasisvmbtesting_datagrabber.py:10]: I see your point, I'll update the docstrings.

**Comment by LeSasse** [comment]
> > I think there also some changes that could be applied in the tests of the datagrabber module. For example, under `junifer/datagrabber/aomic/tests` in `test_id1000.py` there are some functions like `test_id1000_datagrabber`. Should they be renamed as well? I am not sure since in that instance I guess they don't refer to a class name explicitly, but might be worth reviewing that sub-package. (example here https://github.com/juaml/junifer/blob/main/junifer/datagrabber/aomic/tests/test_id1000.py#L13)
> 
> I did think about that and that's a fair point. I intentionally kept this PR to `junifer.testing` and I'll revisit the other ones in a separate PR. :D

In that case, I think I have nothing more to add. LGTM

**Review by LeSasse — APPROVED** [review]

**Pull request merged**